_episodes/01-introduction.md

```diff
@@ -92,32 +92,36 @@ by different publishers and under different licences.
 
 ## Import
 
-1. Download the CSV files from
+To import data, you'll need to open DB Browser for SQLite and download a zip file containing the data files for this tutorial.
```

Comment:
The presence of the "and" make this sound like the DB Browser does the downloading.  Are they to download with a web browser? If so, maybe make this two sentences. And maybe I'd put the downloading of the data as the first step.